src/sonic-yang-models/tests/yang_model_tests/tests_config/neigh.json

```diff
@@ -56,18 +56,73 @@
             "sonic-neigh:NEIGH": {
                 "NEIGH_LIST": [
                     {
-                        "vlan": "INVALIDVlan",
+                        "port": "INVALIDVlan",
                         "neighbor": "100.1.1.3",
                         "neigh": "00:02:02:03:04:05",
                         "family": "IPv4"
                     },
                     {
-                        "vlan": "Vlan1000",
+                        "port": "Vlan1000",
                         "neighbor": "100.1.1.4",
                         "family": "IPv4"
                     }
                 ]
             }
         }
+    },
+
+    "VALID_NEIGH_PORTCHANNEL": {
+        "sonic-neigh:sonic-neigh": {
+            "sonic-neigh:NEIGH": {
+                "NEIGH_LIST": [
+                    {
+                        "port": "PortChannel11",
+                        "neighbor": "100.1.1.3",
+                        "neigh": "00:02:02:03:04:05",
+                        "family": "IPv4"
+                    },
+                    {
+                        "port": "PortChannel11",
+                        "neighbor": "100.1.1.4",
+                        "family": "IPv4"
+                    }
+                ]
+            }
+        },
+        "sonic-portchannel:sonic-portchannel": {
+            "sonic-portchannel:PORTCHANNEL": {
+                "PORTCHANNEL_LIST": [
+                    {
+                        "admin_status": "up",
+                        "name": "PortChannel11"
+                    }
+                ]
+            }
+        }
+    },
+
+    "NEIGH_INVALID_PORTCHANNEL": {
+        "sonic-neigh:sonic-neigh": {
+            "sonic-neigh:NEIGH": {
+                "NEIGH_LIST": [
+                    {
+                        "port": "PortChannel10",
```

Comment:
Could you add one more example for PortChannel1024 to validate the portchannel name length?